VoteCounter/src/VoteCount.cs

```diff
@@ -57,12 +63,40 @@ private void findVotes(Post post)
             {
                 if (isVoteValid(match.Groups[1].Value))
                 {
+                    
+                    string foundPlayer = new string('a',1);
```

Comment:
I think that might be because you used new where you don't need to: you can just string foundPlayer = "";